chapter_preface/index_vn.md

```diff
@@ -600,14 +600,33 @@ And if you have not used Python before,
 you may want to peruse this [Python tutorial](http://learnpython.org/).
 -->
 
-*dịch đoạn phía trên*
+Cuốn sách này dành cho các bạn sinh viên (đại học hoặc sau đại học),
+các kỹ sư và các nhà nghiên cứu, những người tìm kiếm một nền tảng vững chắc
+về những kỹ thuật thực tế của học sâu.
+Bởi vì chúng tôi giải thích mọi khái niệm từ đầu,
+bạn không bắt buộc phải có nền tảng về học sâu hay học máy.
+Việc giải thích đầy đủ các phương pháp học sâu
+đòi hỏi một số kiến thức về toán học và lập trình,
+nhưng chúng tôi sẽ chỉ giả định rằng bạn nắm được một số kiến thức cơ bản
+về đại số tuyến tính, giải tích, xác suất,
+và lập trình Python.
+Hơn nữa, trong phần Phụ lục, chúng tôi cung cấp thêm
+về hầu hết các phần toán được đề cập trong cuốn sách này.
+Phần lớn thời gian, chúng tôi sẽ ưu tiên giải thích trực quan và các ý tưởng
+hơn là giải thích chặt chẽ bằng toán.
+Có rất nhiều cuốn sách tuyệt vời có thể thu hút bạn đọc quan tâm sâu hơn nữa.
+Chẳng hạn, cuốn "Phân tích tuyến tính" của Bela Bollobas :cite:`Bollobas.1999`
+bao gồm cả đại số tuyến tính và phân tích hàm chức năng ở mức độ rất chi tiết.
+Cuốn "Tất cả về số liệu thống kê" :cite:`Wasserman.2013` là hướng dẫn tuyệt vời để học thống kê.
```

Comment:
```suggestion
Cuốn "All of Statistics" :cite:`Wasserman.2013` là hướng dẫn tuyệt vời để học thống kê.
```
Theo mình cuốn "All of Statistics" chưa có bản dịch tiếng Việt, mình nghĩ nên giữ nguyên tên sách.